docs/docs/gatsby-i18n.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Gatsby i18n
+---
+Serving users content in a way that is adapted to their language & culture is part of a great user experience. When you make an effort to adapt web content to a users location that practice is called internationalization (i18n).
+
+In practice i18n involves translating text and formatting dates, numbers, and strings based on the user's detected location. For example, a date displayed for a user in the United States would follow the mm/dd/year date format, but for a user in the UK the date format would change to dd/mm/year.
+
+This guide is a brief look at the options that exist for enhancing your Gatsby project for internationalization.
+
+### Choosing a package
```

Comment:
Can you generalize some of this away from your specific project use case? "We opted for", "we wanted". Would make sense in a blog post, but for a doc it makes more sense to be generalized.

Reply:
Changed it to be much more general - if it's a little too much please let me know.